xmppserver/src/main/webapp/server-session-details.jsp

```diff
@@ -104,7 +109,7 @@
             if (!inSessions.isEmpty()) { %>
                 <%= inSessions.get(0).getHostAddress() %>
                 /
-                <%= inSessions.get(0).getHostName() %>
+                <%= inSessions.get(0).getHostName() %> 
```

Comment:
There's a mysterious whitespace being added to the end of this line. Probably unneeded. :)